packages/components/src/Button.tsx

```diff
@@ -0,0 +1,27 @@
+import type { ButtonVariants } from './styles/Button.css';
+import type { ForwardedRef } from 'react';
+import type { ButtonProps as AriaButtonProps } from 'react-aria-components';
+
+import { clsx } from 'clsx';
+import { forwardRef } from 'react';
+import { Button as AriaButton } from 'react-aria-components';
+
+import { variants } from './styles/Button.css';
+
+type ButtonProps = AriaButtonProps & ButtonVariants;
```

Comment:
Oh, in launchpad-experimental I removed `disabled` to avoid confusing with `isDisabled` which is how RAC controls that state.

Curious what you think about that.

Reply:
Yeah same here. We should keep additional props to a minimum, and since folks can iteratively adopt we don't need to shy away from breaking changes/prop cleanup.